packages/events/src/service/locations/locations.ts

```diff
@@ -53,16 +40,19 @@ export async function syncLocations() {
   return setLocations(locations)
 }
 
-export const getLocations = async () => {
-  const locations = await locationsRepo.getLocations()
+/**
+ * NOTE: Be cautious when calling this function as it fetches all locations from the database.
+ * Do you really need all of them? Consider using more specific functions if possible. Act as if there could be hundreds of thousands of locations.
```

Comment:
Wonder if we should add hundreds of thousands of demo data to keep things in rail